.github/templates/common.yml.j2

```diff
@@ -67,8 +67,11 @@ concurrency:
         env:
           ALPINE_IMAGE: "308535385114.dkr.ecr.us-east-1.amazonaws.com/tool/alpine"
         run: |
+          retry () {
+              $*  || (sleep 1 && $*) || (sleep 2 && $*)
+          }
           # Ensure the working directory gets chowned back to the current user
-          docker run --rm -v "$(pwd)":/v -w /v "${ALPINE_IMAGE}" chown -R "$(id -u):$(id -g)" .
+          retry docker run --rm -v "$(pwd)":/v -w /v "${ALPINE_IMAGE}" chown -R "$(id -u):$(id -g)" .
```

Comment:
Actually, it's quite bad to retry the run, can you please split it into `retry docker pull "${ALPINE_IMAGE}"` and then `docker run` without the retry?